### ISSUE TITLE
Neural Style Transfer Tutorial typo

### ISSUE THREAD

**Issue by Marceliny** [pytorch/tutorials#870]
## 📚 Documentation

In style transfer tutorial, under the [gradient descent](https://pytorch.org/tutorials/advanced/neural_style_tutorial.html#gradient-descent) section, second paragraph's last sentence has a typo that 'module' is written as 'modul'.

The last sentence, 
> The optimizer requires a “closure” function, which reevaluates the modul and returns the loss.

needs to be 
> The optimizer requires a “closure” function, which reevaluates the module and returns the loss.
